bin/kcl-bootstrap

```diff
@@ -147,14 +151,10 @@ function parseArguments() {
 }
 
 function startKinesisClientLibraryApplication(options) {
-  var classpath = '-cp ' + getClasspath(options).join(getPathDelimiter());
+  var classpath = getClasspath(options).join(getPathDelimiter());
   var java = options.java;
-  var propertiesFile = '--properties-file ' + options.properties;
-  var logConfiguration = '';
-  if (options.logConfiguration) {
-    logConfiguration = '--log-configuration ' + options.logConfiguration;
-  }
-  var args = [classpath, MULTI_LANG_DAEMON_CLASS, propertiesFile, logConfiguration];
+  var logConfiguration = options.logConfiguration ? ['--log-configuration', options.logConfiguration] : [];
+  var args = ['-cp', classpath, MULTI_LANG_DAEMON_CLASS, '--properties-file', options.properties, ...logConfiguration];
```

Comment:
this line seems to be necessary on mac as well

Reply:
> this line seems to be necessary on mac as well

Confirmed

Reply:
And linux too